FirstDeliverable: V-1.1.0

ponchOlvera opened the pull request:
+ Added implementation for ListAdapter into the CharacterRecyclerViewAdapter.kt
+ Removed unused  dependencies.

ponchOlvera committed:
FirstDeliverable: V-1.1.0

+ Added implementation for ListAdapter into the CharacterRecyclerViewAdapter.kt
+ Removed unused  dependencies.

pestrada commented on app/src/main/java/com/wizeline/heroes/ui/adapters/CharacterRecyclerViewAdapter.kt:
```diff
@@ -19,17 +20,31 @@ class CharacterRecyclerViewAdapter(
+
+    private class DiffCallback : DiffUtil.ItemCallback<Character>() {
+        override fun areItemsTheSame(oldItem: Character, newItem: Character): Boolean {
+            return oldItem.name == newItem.name
```
I would compare the items with the 'id' instead of name. Because that seems to be the unique identifier.

ponchOlvera replied:
Done.

pestrada reviewed (approved):
Left one minor change. Feel free to merge once that's fixed. Good job🎉👍

ponchOlvera committed:
FirstDeliverable: V-1.1.1

+ Added private variables.
+ Change list adapter validation and associate it with id instead of name.
